pinc/faq.inc

```diff
@@ -220,7 +219,5 @@ function maybe_redirect_to_external_faq($langcode = null)
         $url = get_external_faq($faq, "en");
     }
 
-    if ($url) {
-        metarefresh(0, $url, '', '', true);
-    }
+    metarefresh(0, $url, '', '', true);
```

Comment:
I didn't add an `assert($url !== null)` as those could be prod impacting, depending on our config.